src/init/features/functions/typescript.js

```diff
@@ -6,6 +6,7 @@ var path = require("path");
 
 var npmDependencies = require("./npm-dependencies");
 var { prompt } = require("../../../prompt");
+var utils = require("../../../utils");
```

Comment:
Since we are only using `getNodeVersionString`, I think it make sense to do something like

```
var { getNodeVersionString}  = require("../../../utils");
```

Reply:
I intend to use `logWarning()` from `utils` to warn users of deprecated Node versions (like this code below):

https://github.com/firebase/firebase-tools/blob/f413eb9eb6940ee20dea748b18bb25ce185e7d7f/src/accountImporter.js#L163-L166

In that case, should I keep it that way?